### ISSUE TITLE
Remove hidden edit link in main content listing

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#9821]
## Background

During an accessibility audit, it was found that a hidden link is appearing in the main content listing beneath the edit button. This second, visually hidden link takes the user to the editor screen. This link does not have an accessible name so when it is focused on, it is unclear what the intention of the link is. The accessibility lead suggests that the visually hidden link does not serve a purpose and should be removed from the DOM.

Example of hidden link:

<img width="744" alt="Screen Shot 2022-07-17 at 9 08 20 PM" src="https://user-images.githubusercontent.com/90111970/179435788-5bd95016-13ac-4308-92f5-3339664344bb.png">

## Accessibility Standard

WCAG version 2.0 AAA, [Criterion 2.4.9](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)

## Acceptance Criteria

- [ ] tech review and removal of the hidden link

**Comment by EWashb** [comment]
@edmund-dunn do you have any understanding of the level of effort on this? Is there something more to this issue or is it truly a regression or bug?

Btw- this is not priority for sprint 65, just trying to determine stretch items/sprint 66.

**Comment by edmund-dunn** [comment]
@EWashb this appears to be an issue with Views. It is outputting two links. Finding the source of the issue could take some time, but I would say it is a bug.
